test/job.ts

```diff
@@ -357,15 +357,16 @@ describe('BigQuery/Job', () => {
       const error = new Error('Error.');
       const apiResponse = {
         status: {
-          errors: error,
+          errorResult: error,
+          errors: [error],
         },
       };
 
       const sandbox = sinon.createSandbox();
 
       beforeEach(() => {
         job.getMetadata = (callback: Function) => {
-          callback(null, apiResponse, apiResponse);
+          callback(null, apiResponse);
```

Comment:
nit: it would be cool if we could get a test case around this change in behavior.

Reply:
I actually thought this was a behavior change that wasn't pulled over to BigQuery correctly, but now that you said this I think we might have stumbled on to a bug in `nodejs-common`

Reply:
Oh, nevermind, my initial guess was right. There was a breaking change in common `0.28.0` and the fix apparently never found its way here.